src/shared/components/loader/Loader.tsx

```diff
@@ -1,50 +1,20 @@
-import styled, { keyframes } from 'styled-components';
-import { variant as styledSystemVariant } from 'styled-system';
+import cn from 'classnames';
 
-type LoaderProps = React.ComponentPropsWithRef<typeof StyledLoader> & {
+type LoaderProps = React.ComponentPropsWithRef<'span'> & {
     variant: LoaderVariant;
 };
 
 type LoaderVariant = 'small' | 'big' | 'warning';
 
 export const Loader = ({ variant, ...rest }: LoaderProps) => {
-    return <StyledLoader variant={variant} {...rest} />;
+    return (
+        <span
+            className={cn('rounded-full animate-spin', {
```

Comment:
```suggestion
            className={cn('rounded-full animate-spin border-solid', {
```
since all options have a border